parser/json/value/value.go

```diff
@@ -128,12 +130,12 @@ func (v *Value) String() (string, error) {
 }
 
 // MustString returns a string value or panics if the value type is not string.
-func (v *Value) MustString() string {
+func (v *Value) MustString() []byte {
```

Comment:
I feel like we should leave this as `MustString` since we're consistent w/ naming everything `String` w/ the assumption that the underlying data is `[]byte`. If we rename this, I feel like we'd need to rename everything to be consistent.

Reply:
I think we should rename everything to `Bytes` when the data is `bytes`, other wise there will be a big confusion in the future when you call a `getString()` but got `bytes` instead. The reason these functions and variables were named `xxxString` is because they were strings and now they are not, so the name should reflect the change as well.